Moo/play/lib.py

```diff
@@ -354,9 +347,9 @@ def index(config, sort=None):
     return sorted(albums, key=sort_key, reverse=True)
 
 
-def album(track, metadata):
+def album_info(track, metadata):
     '''
-    Changed def info to def album()
+    Changed def info to def album_info()
```

Comment:
Please remove this from the docstring. Git will show us this change. Comments like this should be added as a [``-m <msg>``](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--mltmsggt) to your commit.